src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Product/_singleBox.html.twig

```diff
@@ -2,6 +2,9 @@
     <a href="{{ path(product) }}"><h4>{{ product.name|truncate(18) }}</h4></a>
     <a href="{{ path(product) }}">
         <img class="img-rounded img-responsive" src="{{ product.image ? product.image.path|imagine_filter('sylius_medium') : 'http://placehold.it/262x255' }}" alt="{{ product.name }}" />
+        {% if (product.masterVariant.crossedprice != 0) %}
```

Comment:
I would put it inside the main span so that it could be used as a macro something like:

```
<span class="label label-primary">
    {{ sylius_customer_product_price(variant) }}
</span>
```

Again not sure about naming, could be formatted_product_price()?
Then your markup might look something like:

```
<span class="label label-primary">
    <span class="product-price"><span class="crossed">€40<span>€25</span>
</span>
```

Then you have something reusable and don't have to repeat the formatting that you've got below - do it once in the macro.